src/_includes/_layouts/About.jsx

```diff
@@ -0,0 +1,194 @@
+                  linkedin={"https://www.linkedin.com/in/capostol/"}
+                />
+              }
+              {
+                <comp.MemberCard
+                  key={2}
+                  name={"Miguel Merlin"}
+                  role={"VP Technology"}
+                  image_url={"../assets/eboard/vp_tech.jpg"}
+                  linkedin={
+                    "https://www.linkedin.com/in/miguel-angel-merlin-arriola/"
+                  }
+                />
+              }
+              {
+                <comp.MemberCard
+                  key={3}
+                  name={"Ezri Zhu"}
+                  role={"VP Technology"}
+                  image_url={"../assets/eboard/vp_tech_1.png"}
+                  linkedin={"https://www.linkedin.com/in/~ezri/"}
+                />
+              }
+              {
+                <comp.MemberCard
+                  key={4}
+                  name={"Audrey Yoo"}
+                  role={"VP Design"}
+                  image_url={"../assets/eboard/vp_design.png"}
+                  linkedin={"https://www.linkedin.com/in/audrey-june-yoo/"}
+                />
+              }
+              {
+                <comp.MemberCard
+                  key={5}
+                  name={"Sahana Ali"}
+                  role={"VP Design"}
+                  image_url={"../assets/eboard/vp_design_1.png"}
+                  linkedin={"https://www.linkedin.com/in/sahanaali/"}
+                />
+              }
+              {
+                <comp.MemberCard
+                  key={6}
+                  name={"Massimo Spina"}
+                  role={"VP Finance"}
+                  image_url={"../assets/eboard/vp_finance.png"}
+                  linkedin={
+                    "https://www.linkedin.com/in/massimo-spina-b47b12284/"
+                  }
+                />
+              }
+              {
+                <comp.MemberCard
+                  key={7}
+                  name={"Justin Ferber"}
+                  role={"VP Projects"}
+                  image_url={"../assets/eboard/vp_projects.png"}
+                  linkedin={
+                    "https://www.linkedin.com/in/massimo-spina-b47b12284/"
+                  }
+                />
+              }
+              {
+                <comp.MemberCard
+                  key={8}
+                  name={"Shawn Aviles"}
+                  role={"Alumni Chair"}
+                  image_url={"../assets/eboard/alumni_chair.png"}
+                  linkedin={
+                    "https://www.linkedin.com/in/massimo-spina-b47b12284/"
+                  }
+                />
+              }
+              {
+                <comp.MemberCard
+                  key={9}
+                  name={"Emilio Cardillo"}
+                  role={"Freshman Rep"}
+                  image_url={"../assets/eboard/freshman_rep.png"}
+                  linkedin={
+                    "https://www.linkedin.com/in/massimo-spina-b47b12284/"
```

Comment:
Emilio's LinkedIn: https://www.linkedin.com/in/emilio-cardillo-schrader-6594b7308/